src/call.c

```diff
@@ -82,6 +82,7 @@ struct call {
 
 	uint32_t rtp_timeout_ms;  /**< RTP Timeout in [ms]                  */
 	uint32_t linenum;         /**< Line number from 1 to N              */
+	struct list *custom_hdrs; /**< List of custom headers if any */
```

Comment:
is this for the outgoing call or the incoming call ?

Reply:
This is for outgoing calls. It is using in `ua_connect`.